consensus/polybft/polybft_config.go

```diff
@@ -104,6 +104,8 @@ type BridgeConfig struct {
 	StakeManagerAddr                  types.Address `json:"stakeManagerAddr"`
 	// only populated if stake-manager-deploy command is executed, and used for e2e tests
 	StakeTokenAddr types.Address `json:"stakeTokenAddr,omitempty"`
+	BLSAddress     types.Address `json:"blsAddr"`
+	BN256G2Address types.Address `json:"bn256G2Addr"`
```

Comment:
Why do you need these two addresses in the genesis? Were you thinking to put it just in case somebody is curios about them (since Edge is not using them)...

Reply:
Because they are deployed in the `deploy` command, but used for initialization of `CheckpointManager` in `supernet` command. We weren't saving them to `genesis` before, but now we need to.